week_4_analytics_engineering/README.md

```diff
@@ -61,6 +61,12 @@ After local installation you will have to set up the connection to PG in the `pr
 
 _Note: This video is shown entirely on dbt cloud IDE but the same steps can be followed locally on the IDE of your choice_
 
+_Errata: If you recieve an error stating "Permission denied while globbing file pattern." while trying to run the fact_trips model in dbt this video may helpful for resolving the issue._
```

Comment:
Thanks for doing this! Sorry for my delay, looks like I missed the notification. 
I would avoid using Errata as I don't consider this an error in what this video shows but rather a misconfiguration in BQ. Can we use _note_ instead as the other comments in the docs? 
I think it would also fit best in the config part in the first section (where the quick hack is linked) or the dbt_cloud_setup in the section 1 where the BQ permissions are defined and I added a note for this error there as well .

Reply:
I'm having a bit of trouble re-basing my repo for this. I'll give it another shot this evening and if that does not work I will re-fork the DataTalksClub repo and do a new pull request. If it goes that way I will post a note here as well.